client/src/connection/itc/LineParser.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2024, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export class LineParser {
+  protected errorLines: string[] = [];
+  protected capturingError: boolean = false;
+  protected startTag: string;
+  protected endTag: string;
+
+  public constructor(startTag: string, endTag: string) {
+    this.startTag = startTag;
+    this.endTag = endTag;
+  }
```

Comment:
nit: just a note that this can be simplified to
`public constructor(protected startTag: string, protected endTag: string) {}`
https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties
But I'm fine if you prefer this

Reply:
I updated it to this format in https://github.com/sassoftware/vscode-sas-extension/pull/842/commits/d7f28f15d5c4ec6925932a8f3ac546b28e816ecd . That said, I'm not entirely sure how I feel about it. 

I think my reservation with this approach is that it requires removing the class attributes from the top of the class. For me, this makes it a little less clear what attributes are defined. That said, it's super minor and as long as the constructor is close enough to the rest of the attributes, it seems fine